subsys/net/lib/dns/dns_pack.h

```diff
@@ -328,4 +329,38 @@ int dns_unpack_response_query(struct dns_msg_t *dns_msg);
 int dns_copy_qname(u8_t *buf, u16_t *len, u16_t size,
 		   struct dns_msg_t *dns_msg, u16_t pos);
 
+/**
+ * Unpacks the DNS query.
+ *
+ * @param msg Structure containing the response.
+ * @param src_id Transaction id, this is returned to the caller.
+ * @retval 0 on success, <0 if error
+ * @retval -ENOMEM if the buffer in msg has no enough space to store the header.
+ * The header is always 12 bytes length.
```

Comment:
can you align the to previous @<keyword> +1 space, relevantly ?
It won't change anything in doc generation, but in-place reading will be easier. And afaik we did so everywhere.

Apply this on all this patch

Reply:
That could be done in separate patch, all the documentation in this file is now looking uniform.